src/Text/Pandoc/Writers/Haddock.hs

```diff
@@ -209,6 +209,10 @@ inlineToHaddock opts (Span (ident,_,_) ils) = do
 inlineToHaddock opts (Emph lst) = do
   contents <- inlineListToHaddock opts lst
   return $ "/" <> contents <> "/"
+-- Underline is not supported
+inlineToHaddock opts (Underline lst) = do
+  contents <- inlineListToHaddock opts lst
+  return contents
```

Comment:
Would it make sense to render as Emphasis here?  I'm not sure.

Reply:
Done.